pkgs/top-level/all-packages.nix

```diff
@@ -13615,6 +13615,8 @@ in
 
   lyx = callPackage ../applications/misc/lyx { };
 
+  inherit (pythonPackages) magic-wormhole;
```

Comment:
Would it work if I changed it to `inherit (pythonApplications) magic-wormhole;`, or something equivalent?